docs/cubeviz/displayspectra.rst

```diff
@@ -3,11 +3,12 @@ Displaying Spectra
 ******************
 
 A collapsed spectrum of the cube displayed in the upper-left viewer
-automatically appears in the 1D spectrum viewer, using the Maximum
-collapse method.  The collapse method can be changed in the :guilabel:`Viewer`
-tab of the |icon-settings-sliders| icon in the spectrum viewer. Additional spectra
+automatically appears in the 1D spectrum viewer, using the set collapse function.
+The collapse function can be changed in the :guilabel:`Line`
+tab of the Plot Options plugin, which can be reached by clicking the |icon-settings-sliders|
+icon in the spectrum viewer. Additional spectra
 can be loaded into the spectrum viewer, as detailed in the linked documentation
-below. 
+below.
```

Comment:
we should make sure to update this again with the current effort to replace this (and direct the user to the spectral extraction plugin instead).

Reply:
Absolutely!